{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2019-19844",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "Django before 1.11.27, 2.x before 2.2.9, and 3.x before 3.0.1 allows account takeover. A suitably crafted email address (that is equal to an existing user's email address after case transformation of Unicode characters) would allow an attacker to be sent a password reset token for the matched user account. (One mitigation in the new releases is to send password reset tokens only to the registered user email address.)"
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://docs.djangoproject.com/en/dev/releases/security/",
                "refsource": "MISC",
                "url": "https://docs.djangoproject.com/en/dev/releases/security/"
            },
            {
                "name": "https://groups.google.com/forum/#!topic/django-announce/3oaB2rVH3a0",
                "refsource": "MISC",
                "url": "https://groups.google.com/forum/#!topic/django-announce/3oaB2rVH3a0"
            },
            {
                "name": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/",
                "refsource": "CONFIRM",
                "url": "https://www.djangoproject.com/weblog/2019/dec/18/security-releases/"
            },
            {
                "name": "USN-4224-1",
                "refsource": "UBUNTU",
                "url": "https://usn.ubuntu.com/4224-1/"
            },
            {
                "name": "DSA-4598",
                "refsource": "DEBIAN",
                "url": "https://www.debian.org/security/2020/dsa-4598"
            },
            {
                "name": "20200108 [SECURITY] [DSA 4598-1] python-django security update",
                "refsource": "BUGTRAQ",
                "url": "https://seclists.org/bugtraq/2020/Jan/9"
            },
            {
                "name": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html",
                "refsource": "MISC",
                "url": "http://packetstormsecurity.com/files/155872/Django-Account-Hijack.html"
            },
            {
                "name": "https://security.netapp.com/advisory/ntap-20200110-0003/",
                "refsource": "CONFIRM",
                "url": "https://security.netapp.com/advisory/ntap-20200110-0003/"
            },
            {
                "name": "FEDORA-2020-adb4f0143a",
                "refsource": "FEDORA",
                "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HCM2DPUI7TOZWN4A6JFQFUVQ2XGE7GUD/"
            },
            {
                "name": "GLSA-202004-17",
                "refsource": "GENTOO",
                "url": "https://security.gentoo.org/glsa/202004-17"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}